.travis.yml

```diff
@@ -4,8 +4,6 @@ matrix:
     - rvm: 2.4.1 # Test web AND cli with specific version
       gemfile: Gemfile
     # Otherwise test only the cli
-    - rvm: 1.9
```

Comment:
Any particular reason why you removed this?

Reply:
Because ruby 1.9 is no more supported https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/

I m pretty sur it is because since your last merge (upgrade gem) the 1.9 test stopped working because of gem incompatible with r 1.9

Reply:
Fair enough :-)